addons/account-settings-capitalize/addon.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "Capitalized Account Settings",
+  "name": "Capitalize Account Settings",
```

I don't think this needs to be changed as it was already a noun.

---

I think I added a single comment instead of starting a review, wait for more comments...

---

It's also an Easter egg so i don't necessarily think it's worth changing either.

---

It still makes the same amount of sense, and it's shorter.

The question is, are we okay with it being a phase like it now is?

---

What about
```suggestion
  "name": "Account Settings Capitalization",
```
It is longer though

---

I think "Account Settings" needs to be in quotation marks to make this clearer.